packages/menu/test/submenu.test.ts

```diff
@@ -879,6 +695,7 @@ describe('Submenu', () => {
         it('closes back to the first overlay without a `root` when clicking away', async function () {
             const rootMenu1 = this.el.querySelector('#submenu-item-1') as Menu;
             const childMenu2 = this.el.querySelector('#submenu-item-2') as Menu;
+            document.body.style.setProperty('padding', '50px');
```

Comment:
Just for me to learn - what purpose does the padding serve here? Is it to have a space to mouse to when removing focus?

Reply:
Yep